core/src/labels/labelSet.cpp

```diff
@@ -0,0 +1,21 @@
+#include "labelSet.h"
+
+namespace Tangram {
+
+LabelSet::~LabelSet() {}
+
+void LabelSet::reset() {
+    for (auto& label : m_labels) {
+        label->resetState();
+    }
+}
+
+void LabelSet::setLabels(std::vector<std::unique_ptr<Label>>& _labels) {
+    typedef std::vector<std::unique_ptr<Label>>::iterator iter_t;
+    m_labels.insert(m_labels.begin(),
```

Comment:
should this be `m_labels.end()`? You want to retain the labels previously added by the styleBuilder right?